torchgeo/datasets/levircd.py

```diff
@@ -262,3 +184,204 @@ def get_rgb(img: Tensor) -> "np.typing.NDArray[np.uint8]":
             plt.suptitle(suptitle)
 
         return fig
+
+    @abc.abstractmethod
+    def _load_files(self, root: str, split: str) -> list[dict[str, str]]:
+        """Return the paths of the files in the dataset.
+
+        Args:
+            root: root dir of dataset
+            split: subset of dataset, one of [train, test]
+
+        Returns:
+            list of dicts containing paths for each pair of image1, image2, mask
+        """
+
+    @abc.abstractmethod
+    def _check_integrity(self) -> bool:
+        """Checks the integrity of the dataset structure.
+
+        Returns:
+            True if the dataset directories and split files are found, else False
+        """
+
+    @abc.abstractmethod
+    def _download(self) -> None:
+        """Download the dataset and extract it."""
+
+
+class LEVIRCD(LEVIRCDBase):
+    """LEVIR-CD dataset.
+
+    The `LEVIR-CD <https://github.com/justchenhao/STANet>`__
+    dataset is a dataset for building change detection.
+
+    Dataset features:
+
+    * image pairs of 20 different urban regions across Texas between 2002-2018
+    * binary change masks representing building change
+    * three spectral bands - RGB
+    * 637 image pairs with 50 cm per pixel resolution (~1024x1024 px)
+
+    Dataset format:
+
+    * images are three-channel pngs
+    * masks are single-channel pngs where no change = 0, change = 255
+
+    Dataset classes:
+
+    1. no change
+    2. change
```

Comment:
Does our `__getitem__` remap from 0, 255 to 0, 1? Should it? We should label the classes with the returned value, not the stored value.